docs/operation/SOP/PM.DWD.5/PM.DWD.5-TroubleShooting.md

```diff
@@ -0,0 +1,90 @@
+# Trouble Shooting
+
+Most processes within the Dragen WGTS DNA Orchestration use AWS Step Functions to manage the workflow.
+We post all Step Function errors to the #alerts-prod slack channel, a Center staff member can
+then click on the offending Step Function link in the slack message to be taken to the AWS Step Functions console to investigate further.
+
+## Analysis Stuck in DRAFT state.
+
+If the analysis is stuck in DRAFT mode, there may be a couple of reasons for this.
+To determine which issue is causing the problem we can head to the [AWS Step Functions Console][aws_step_functions_console_prod]
+in the production account and look for any RUNNING executions in the 'dragen-wgts-dna--populate-draft-data' step function.
+
+If there is a RUNNING execution for this library id, then the issue is likely due to the Step Functions hanging at the Fastq Sync step.
+See the Fastq Sync stuck section below for more information for hanging processes.
+
+If there is no RUNNING execution for this library id, then the issue is likely due to a payload being incomplete.
+See the Payload Mismatch section below for more information on how to resolve this issue.
+
+### Fastq Sync Stuck
+
+The Fastq Sync step uses a heart-beat timeout so if the step function is genuinely stuck, it will eventually time out and fail the execution.
+You may wish to then 'redrive' the execution from the console.
+However, the Fastq Sync step will receive heartbeats if there are any active jobs running on that particular library id.
+It may be the case that the step is waiting for either a fastq job to complete, such as qc stat calculation (usually taking around 20 minutes),
+OR the fastq data is in archive and is in the process of being thawed (which may take several hours).
+
+### Payload Mismatch
+
+If you can find the most recent execution for this library id, then you may be able to look at the Log Group for the 'validate-draft-payload' lambda.
+
+This lambda will let you know how the payload violates the expected schema.
+You may wish to then manually update the payload and generate a new WorkflowRunUpdate draft event as discussed in [SOP 1][sop_1_rel_path]
+
+## Analysis Stuck in READY state
+
+If the analysis is stuck in READY state, then it is likely that the translation from the READY event to the ICAv2 WES event has failed.
```

Comment:
Where would I check that? 
WES step function? Lambda? Is there a WES service troubleshooting guide/sop?

Perfectly fine to say something like: 
" ... then it's likely an issue with the WES manager, please consult the [WES manager docs](...)..."
or if applicable see below.